website/content/en/preview/concepts/scheduling.md

```diff
@@ -117,6 +117,10 @@ spec:
             vpc.amazonaws.com/pod-eni: "1"
 ```
 
+{{% alert title="Note" color="primary" %}}
+Security groups for pods is [currently unsupported for Windows nodes](https://docs.aws.amazon.com/eks/latest/userguide/security-groups-for-pods.html#:~:text=Security%20groups%20for%20Pods%20can%27t%20be%20used%20with%20Windows%20nodes.)
```

Comment:
- Watch out for the fragment identifier. The text match thing is nonstandard (there is an incubating [draft](https://wicg.github.io/scroll-to-text-fragment/)).

- This detail is very specific to a particular cloud provider integration. Is there a better home for this detail? For example, in the AWS EKS documentation?

Reply:
I'm good with keeping the note. It does point to AWS docs. There is a lot we will need to do to the docs when Karpenter supports multiple cloud providers.

Reply:
@sftim Thanks for mentioning about the fragment identifier, have since removed the nonstandard text fragment directive.

As for the comment regarding this information being specific to a cloud provider, I do acknowledge that the note is specific to AWS, but its also worth mentioning that the section in its current state already makes mention of other AWS specific information related to the AWS VPC CNI Plugin.

I am of the opinion we might need a separate issue to review the current state of the docs and come up with new ways of allowing docs to help with cloud specific information. On this note, @chrisnegus thank you for mentioning your feedback regarding the future plans to update the docs to help alleviate this concern.